schema.graphql

```diff
@@ -33,8 +31,18 @@ type Market @entity { # TODO rename to CToken, not market AND ALL OTHERS
     numberOfSuppliers: Int!
     "Reserves stored in the contract"
     reserves: BigDecimal!
-    "Yearly supply rate. With 2104400 blocks per year"
+    "Yearly supply rate. With x blocks per year at 5 s/block"
     supplyRate: BigDecimal!
+    "supply APY"
+    supplyAPY: BigDecimal!
```

Comment:
what is the difference between `supplyRate` and `supplyAPY` (same for borrowRate and borrowAPY). Are both needed if one can be derived from the other?

Reply:
rate is like simple interest used in APY calculation